Source/ARTRealtime.m

```diff
@@ -1666,15 +1655,16 @@ - (void)realtimeTransportRefused:(id<ARTRealtimeTransport>)transport withError:(
     if (error && error.type == ARTRealtimeTransportErrorTypeRefused) {
```

Comment:
Will take a look.

Reply:
It's triggered when one the following [WebSocket close codes](https://datatracker.ietf.org/doc/html/rfc6455#section-7.4.1) is received:

1003:

> indicates that an endpoint is terminating the connection because it has received a type of data it cannot accept (e.g., an endpoint that understands only text data MAY send this if it receives a binary message).

1008:

> indicates that an endpoint is terminating the connection because it has received a message that violates its policy.  This is a generic status code that can be returned when there is no other more suitable status code (e.g., 1003 or 1009) or if there is a need to hide specific details about the policy.